content/modules/redisearch/_index.md

```diff
@@ -7,145 +7,67 @@ categories: ["Modules"]
-
-Applications developed with open source version of RediSearch are 100%
-compatible with RediSearch in Redis Enterprise. The primary difference
-between Enterprise RediSearch and the open source version is the
-enterprise version is fully distributed when indexing and processing
-queries, whereas the open source version is not. To understand this
-better, read more below how RediSearch partitions work with RS's
-sharding.
-
-Indexing with RediSearch: RediSearch provides a highly functional index.
-You can build compound indexes that index multiple fields in documents.
-Text, numeric and geospatial data types are supported.
-
-The indexing is done incrementally. As the documents change, you can
-feed the changes to the index. You can keep the copy of documents that
-are indexed to return them as part of the search results. However, this
-is optional. You can limit the index to only contain the index structure
-and results return the unique docID you feed in to help retrieve the
-results from another database that contains the full document.
-
-Search and Query with RediSearch: For full-text searches, developers can
-customize the fields queries and ranking of the search results. When
-querying, developers can use multiple predicates that query both text,
-numeric, and geospatial fields in one query. Sorting can be done by a
-given field as well and can be limited with an offset for simplified
-result-paging support.
-
-RediSearch is language aware and support the following languages for
-stemming. The following list of languages are currently supported with
-RediSearch: "arabic", "danish", "dutch", "english", "finnish", "french",
-"german", "hungarian", "italian", "norwegian", "portuguese", "romanian",
-"russian", "spanish", "swedish", "tamil", "turkish"
-
-The index specifically supports auto-complete engines with specific
-commands that can provide real-time interactive search suggestions.
+The RediSearch 2.x module is a source-available project that lets you build powerful searches for Redis databases.
+When combined with Redis Enterprise Software (RS), you can use the same RediSearch protocols and commands
+to get a geo-replicated, integrated query, and full-text search over efficient in-memory indexes.
+
+You can build compound indexes that index multiple fields in documents with text, numeric, and geospatial data types.
+With those indexes you can do language-aware fuzzy matching, fast auto-complete, exact phrase matching, numeric filtering, and geo-radius queries.
+
+As the documents in your database change, the index automatically processes these changes to keep your searches updated.
+You can also limit the index to only contain the index structure so that the results return a unique docID.
+
+For full-text searches, developers can customize the field queries and ranking of the search results.
+When querying, developers can use multiple predicates that query both text, numeric, and geospatial fields in one query.
```

Comment:
s/developers/you

We shouldn't change person here. Earlier in this document, we refer to "you" but then suddenly change to "developers". Let's stick with "you".